src/db/data_insertion.py

```diff
@@ -13,16 +18,22 @@ def data_insert(image_name: str, image_date: datetime, image_time: datetime, ini
         image_time (datetime obj): time image was taken
         init_vecs (list): list of decryption keys(strings) for encrypted bounding boxes in image
         bboxes (list): list of bounding boxes, each bounding box containing coordinates, confidence and classification
+        labels (list): defines the classifications for each bounding box
         input_dir (string): image path in client machine
     """
+    
+    """The 3 classifications included in labels are represented as follows:
+    glasses : 0, goggles : 1, neither : 2
+    Glasses classification is currently not being stored in the database so the list is modified before database insertion
+    """
+    labels = [GOGGLES if label == 1 else NEITHER for label in labels]

```

Comment:
here we have 3 classifications which is inconsistent with what was defined above. also they have different values? this is resulting in unnecessary code like this if statement